config/locales/en.yml

```diff
@@ -25,7 +25,10 @@ en:
         property_form:
           attributes:
             property_owned:
-              inclusion: Please select the option that best decribes your clients property ownership
+              inclusion: Please select the option that best describes your clients property ownership
```

Comment:
As you're fixing typos in this line, could you possibly add the missing apostrophe to "client's"?